lib/fastlane/plugin/fueled/helper/fueled_helper.rb

```diff
@@ -204,8 +204,8 @@ def self.calculate_code_coverage_percentage(
             end
           end
         end
-
-        total_code_coverage_percentage = code_coverage_percentage / number_of_files
+        
+        total_code_coverage_percentage = number_of_files > 0 ? code_coverage_percentage / number_of_files : 100
```

Comment:
@harishsaini  I think we should show a message here in case there are no files, so the user can take another look at the config file and see if he specified the needed files correctly.

An early exit and then showing a message:
`UI.message("No files found")` or something similar.

It's not correct in my opinion to assume that the code coverage percentage is `100%` when you don't actually check the code coverage.

Reply:
@hadiidbouk  I noticed the issue after creating this PR https://github.com/Fueled/project-template-ios/pull/113, 

Tests fail there saying `divide by zero` error. 

We don't have any ViewModel tests there and number_of_files become zero, we do want that PR to pass, what do you suggest we do here, failing tests with a message will not let us merge that PR? 

and it makes sense to me that zero files were tested with zero tests, reaching 100% coverage. it also print the same message 
```
Checked code coverage on 0 file with total percentage of 100%
```

Reply:
yes it should ideally but in reality it fails with divide by zero error since no viewModel files were found and `number_of_files` was zero. you can check the error on this PR https://github.com/Fueled/project-template-ios/pull/113. I've reverted the pluging file there to not use this branch.

Reply:
I am all in for showing a message but we do need to handle the edge case mentioned above and merge my PR [on](https://github.com/Fueled/project-template-ios/pull/113), I can't think of any other solution, please let me know if you can.

Reply:
Just catching up- from what I hear, the suggestion is to exit the coverage check once the `number_of_files` has been calculated and found to be `0`?

Somewhat tangential, but is this the best way to determine what test coverage is? Based on the iOS `.fueled_code_coverage.json` we are simply looking for the number of files that include the file name `ViewModel`. I'm wondering if we might be able to leverage Xcode's code coverage feature, and maybe utilize [tools like this one](https://github.com/fastlane-community/xcov) (at least for iOS cc @hadiidbouk  @harishsaini)

Reply:
I agree with using xcode code coverage tool. Separate work though of course.

Reply:
@hadiidbouk What do you suggest we can do here to move forward? should we merge this one or any other solution you wanna try?

Reply:
For simplicity, we can merge this one, but can you check if you can quickly show a warning here?
The idea is we want to inform the engineer that this script hasn't detected the files (from the targets) mentioned in the config files; so the engineer can go and sort it out.

Reply:
sure, done 👍